Uf 3266 add rule for lost parking permit

foobar-1337 opened the pull request.

josc04 commented on src/main/java/se/sundsvall/processengine/parkingpermit/integration/citizen/model/Address.java:
```diff
@@ -2,55 +2,54 @@
 package se.sundsvall.processengine.parkingpermit.integration.citizen.model;
```
Create model from yaml instead?

foobar-1337 replied:
Fixed

foobar-1337 commented on src/main/java/se/sundsvall/processengine/parkingpermit/ParkingPermitApplication.java:
```diff
@@ -1,15 +1,15 @@
 
 @ServiceApplication
 @EnableFeignClients
 public class ParkingPermitApplication {
```
Fixed

foobar-1337 commented:
Acted on review comments, refactored code and added unit-tests.
The application is now building OK with "mvn clean verify", please keep it that way.... :-)

foobar-1337 commented on src/main/java/se/sundsvall/processengine/parkingpermit/service/MessagingService.java:
```diff
@@ -1,19 +1,18 @@
+import se.sundsvall.processengine.parkingpermit.integration.messaging.MessagingClient;
 
 @Service
 public class MessagingService {
```
I just closed my eyes. This can be removed, but I didn't do it in this PR. 
I can't see any references to this class in my branch. But it's possbile that a reference exists in another branch and this will lead to merge-work later on? 

Maybe refactor later on?

foobar-1337 commented on pom.xml:
```diff
@@ -31,18 +30,10 @@
 			<groupId>se.sundsvall.dept44</groupId>
 			<artifactId>dept44-starter-feign</artifactId>
 		</dependency>
-		<dependency>
```
🥇
